bindings/yarp.i

```diff
@@ -443,6 +443,10 @@ MAKE_COMMS(Bottle)
 %include <yarp/dev/IPositionDirect.h>
 %include <yarp/dev/MultipleAnalogSensorsInterfaces.h>
 
+inline %{
```

Comment:
Hi @nunoguedelha,
Is there any reason why you add these lines here, rather than [here](https://github.com/robotology/yarp/blob/master/bindings/yarp.i#L574) as I suggested (https://github.com/robotology/yarp/issues/1826#issuecomment-411063483). 
I ask since we still get `TypeError: ... of type 'yarp::sig::Vector const &` with this fix but not for the other position.

Reply:
ouch, it's a typo! Sorry for that.

Reply:
one-mil euro typo :smile: (kidding)

Reply:
Actually, looking at it again, I recall that this `%inline` primitive is twofold (http://www.swig.org/Doc1.3/SWIG.html#SWIG_nn43ç):
> The %inline directive inserts all of the code that follows verbatim into the header portion of an interface file. The code is then parsed by both the SWIG preprocessor and parser.

But further in the same file, the SWIG primitive below overrides the `%inline` one:
`%template(Vector) yarp::sig::VectorOf<double>;`
So it would be cleaner to replace the `%inline` command by:
```
%{
typedef yarp::sig::VectorOf<double> Vector;
%}
```
and group it with the other definitions using `yarp::sig::Vector`: refer to this PR I've just submitted: https://github.com/robotology/yarp/pull/1840